Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOneV3/TrendMicroVisionOneV3.py

```diff
@@ -1775,177 +2204,253 @@ def submit_file_entry_to_sandbox(
-    file_name = file_.get("name")
-    file_path = file_.get("path")
-    archive_pass = args.get(ARCHIVE_PASSWORD)
-    document_pass = args.get(DOCUMENT_PASSWORD)
-    query_params: Dict[Any, Any] = {}
-    headers = {AUTHORIZATION: f"{BEARER} {client.api_key}"}
+    file_name = file_.get(NAME, EMPTY_STRING)
+    file_path = file_.get(PATH, EMPTY_BYTE)
     with open(file_path, "rb") as f:
         contents = f.read()
-    data = {}
-    if document_pass:
-        data["documentPassword"] = base64.b64encode(document_pass.encode(ASCII)).decode(
-            ASCII
-        )
-    if archive_pass:
-        data["archivePassword"] = base64.b64encode(archive_pass.encode(ASCII)).decode(
-            ASCII
-        )
-    files = {"file": (f"{file_name}", contents, "application/octet-stream")}
-    try:
-        result = requests.post(
-            f"{client.base_url}{SUBMIT_FILE_TO_SANDBOX}",
-            params=query_params,
-            headers=headers,
-            data=data,
-            files=files,
-        )
-        result.raise_for_status()
-    except HTTPError as http_err:
-        demisto.error(http_err)
-        return_error(http_err)
-    except Exception as err:
-        demisto.error(err)
-        return_error(err)
-    response = result.json()
+    # Make rest call
+    resp = v1_client.submit_file_to_sandbox(
+        file=contents,
+        file_name=file_name,
+        document_password=document_pass,
+        archive_password=archive_pass,
+        arguments=arguments,
+    )
+    # Check if an error occurred during rest call
+    if _is_pytmv1_error(resp.result_code):
+        return_error(message=f"{unwrap(resp.error).message}", error=str(resp.error))
+    # Add results to message to be sent to the War Room
     message = {
+        "code": 202,
+        "message": resp.result_code,
         "filename": file_name,
-        "entryId": entry,
-        "file_path": file_.get("path", ""),
-        "message": "Success",
-        "code": "Success",
-        "task_id": response.get("id", ""),
-        "digest": response.get("digest", ""),
+        "entry_id": entry,
+        "file_path": file_path,
+        "task_id": unwrap(resp.response).id,
+        "digest": unwrap(resp.response).digest.dict(),
+        "arguments": unwrap(resp.response).arguments,
     }
-    results = CommandResults(
+
+    return CommandResults(
         readable_output=tableToMarkdown(
-            TABLE_SUBMIT_FILE_ENTRY_TO_SANDBOX, message, removeNull=True
+            table_name[FILE_ENTRY_TO_SANDBOX_COMMAND],
+            message,
+            headerTransform=string_to_table_header,
+            removeNull=True,
         ),
         outputs_prefix="VisionOne.Submit_File_Entry_to_Sandbox",
-        outputs_key_field="entryId",
+        outputs_key_field="entry_id",
         outputs=message,
     )
-    return results
 
 
-def add_note(client: Client, args: Dict[str, Any]) -> Union[str, CommandResults]:
+def submit_urls_to_sandbox(
```

Comment:
are this code needed ? it's looks like there is no command defined in the YML using this code.

Reply:
added action details to YML.